css/style.css

```diff
@@ -6,6 +6,10 @@
 	display: block;
 }
 
+#body-public footer.hidden {
+	display: none;
```

Comment:
This is not enough: 
<img width="368" alt="bildschirmfoto 2018-10-02 um 23 55 36" src="https://user-images.githubusercontent.com/245432/46379314-ad559680-c69e-11e8-9702-8bb7cb26b7f6.png">

Reply:
I have tested it again removing and adding the fix and for me it works as expected (footer visible without the fix, footer hidden with the fix).

Note that in your screenshot all the rules come from the server (the `display: none` is the generic one from the server, not the one from `#body-public footer.hidden` in _style.css_). Maybe a cache issue?

Reply:
Maybe also from gallery app or so?